docs/TestingBesuProxyModel.rst

```diff
@@ -40,6 +40,20 @@ To run Besu with Hyperledger Avalon, follow these steps:
 
    If you modify the number of nodes, please update the ``no_proxy`` list
    accordingly.
+   Note that HOME/.docker/config.json (Default Docker configuration file) should
```

Comment:
Proxy config is only needed behind a corporate proxy.  So change "Note that" to "If you are behind a corporate proxy," (or similar wording).